diagnostic.md

```diff
@@ -14,7 +20,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$('ol > li').on()
```

Comment:
This will select all `<li>` elements that are direct descendants of an `<ol>` element and attempt to attach an event handler to them.